lua/autorun/server/sv_cfc_spawnpoint.lua

```diff
@@ -1,18 +1,169 @@
 resource.AddWorkshop( "3114959065" )
 
-local HOOK_CANTOOL = "CFC_Spawnpoint2_BannedTools"
-local bannedTools = {
+CFC_SpawnPoints = CFC_SpawnPoints or {}
+
+CFC_SpawnPoints.COMMANDS = {
+    ["unlinkSpawnPoint"] = {
+        ["!unlinkspawn"] = true,
+        ["!unlinkspawnpoint"] = true
+    },
+    ["unlinkThisSpawnPoint"] = {
+        ["!unlinkthis"] = true
+    }
+}
+
+CFC_SpawnPoints.BANNED_TOOLS = {
     ["nocollideeverything"] = true,
     ["nocollide"] = true,
+    ["material"] = true,
+    ["submaterial"] = true,
+    ["fadingdoor"] = true,
+    ["fading_door"] = true,
+    ["colour"] = true,
 }
-hook.Remove( "CanTool" , HOOK_CANTOOL )
-hook.Add( "CanTool" , HOOK_CANTOOL, function( ply, tr, tool )
+
+local COOLDOWN_ON_PLY_SPAWN = CreateConVar( "cfc_spawnpoints_cooldown_on_ply_spawn", 10, { FCVAR_ARCHIVE }, "When a player spawns, they must wait this many seconds before they can create/link spawn points.", 0, 1000 )
+
+local commands = CFC_SpawnPoints.COMMANDS
+local bannedTools = CFC_SpawnPoints.BANNED_TOOLS
+
+local heightOfSpawnPointPlusOne = 16
```

Comment:
What's this used for? Maybe you could call this `local heightOfWhateverThisIsUsedFor = 15 + 1` or similar